src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,208 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    public MyLinkedList() {
+        size = 0;
```

Comment:
these are default values - so this may be omitted

Reply:
done